hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -995,4 +996,6 @@ EchoRPCResponse echoRPCReq(byte[] payloadReq,
    */
   boolean recoverLease(String volumeName, String bucketName,
                               String keyName) throws IOException;
+
+  ManagedSecretKey getCurrentSecretKey() throws IOException;
```

Comment:
I'm confused about what this API does. Is it supposed to tell OM to fetch the current from SCM? The name and the implementation seems to indicate it only get the current secret key.

Reply:
Yes, the `getCurrentSecretKey()` method under `OzoneManagerProtocol` has two implementations -> `OzoneManager` and `OzoneManagerProtocolClientSideTranslatorPB`. In `OzoneManager`, I am getting the current secret key using `SecretKeySignerClient secretKeyClient`, which has been initialized to:

```
SecretKeyProtocol secretKeyProtocol = HddsServerUtil.getSecretKeyClientForOm(conf);
secretKeyClient = new DefaultSecretKeySignerClient(secretKeyProtocol);
```

Reply:
I made the required changes to avoid the `ManagedSecretKey` import, thanks